src/chat.ts

```diff
@@ -1,33 +1,38 @@
 import { ChatGPTAPI } from 'chatgpt';
+import core from '@actions/core';
+
 export class Chat {
-  private chatAPI: ChatGPTAPI;
+    private chatAPI: ChatGPTAPI;
 
-  constructor(apikey: string) {
-    this.chatAPI = new ChatGPTAPI({
-      apiKey: apikey,
-    });
-  }
+    constructor(apikey: string) {
+        this.chatAPI = new ChatGPTAPI({
+            apiKey: apikey,
+        });
+    }
 
-  private generatePrompt = (patch: string) => {
-    return `Bellow is the code patch, please help me do a brief code review, if any bug risk and improvement suggestion are welcome
+    private generatePrompt = (patch: string) => {
+        return `Bellow is the code patch, please help me do a brief code review, if any bug risk and improvement suggestion are welcome
     ${patch}
     `;
-  };
+    };
 
-  public codeReview = async (patch: string) => {
-    if (!patch) {
-      return '';
-    }
+    public codeReview = async (patch: string) => {
+        if (!patch) {
+            return '';
+        }
+
+        console.time('code-review cost');
 
-    console.time('code-review cost');
-    const prompt = this.generatePrompt(patch);
+        const prompt = this.generatePrompt(patch);
+        const language = core.getInput('language', { required: false });
+        const promptSuffix = language ? `\nlet's start with ${language}` : "\nlet's start";
```

Comment:
Have you tested with this, I just want to confirm that using this prompt GPT will review in the corresponding language?